zokrates_core_test/tests/tests/div.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
It seems like this file uses both tabs and spaces which looks weird

Reply:
Fixed in #1221